### DISCUSSION TITLE
Use a translation platform to simplify contributions from community

### DISCUSSION THREAD

**Discussion by thebiblelover7** [ChurchApps/ChurchAppsSupport#318] [FreeShow] [answered]
I'd like to propose an idea to use a translation platform such as **[Transifex](https://www.transifex.com/open-source/)** or **[Weblate](https://weblate.org/en/hosting/)**. It would make translating a lot easier to manage and a lot easier for newcomers that don't know how to use GitHub, or json files. Both of these services seem to be quite simple to setup and both offer a "_forever free_" plan for open source projects. Transifex is proprietary, but it seems to be more generous. Weblate is open source, but supports only 10,000 strings, and 90 languages in the free plan.

If this sounds like a good idea, I would like to volunteer to help set it up.

**Comment by vassbo** [comment, marked as answer]
I have thought about it, but didn't realize Transifex was free for open-source projects.
I'm not sure about managing it, but I'll see how it goes!
I have created a page here: https://app.transifex.com/nettbiter/freeshow/dashboard/
